openshift/pipelines/Jenkinsfile.cicd

```diff
@@ -22,9 +22,6 @@ pipeline {
   environment {
     CI = 'true'
 
-    // The name of your application
```

Comment:
I thought you just added this value, but it looks like you're removing it now

Reply:
I added the ability to have this value come from the env vars, not just hardcoded.